documents/src/pages/elements/interactive-chart.md

```diff
@@ -488,35 +488,28 @@ A Seasonality chart can be created using multiple series with the same timespan.
 ```javascript
 ::import-elements::
 
-const monthCount = {};
-const months = {
-  1: "January",
-  2: "February",
-  3: "March",
-  4: "April",
-  5: "May",
-  6: "June",
-  7: "July",
-  8: "August",
-  9: "September",
-  10: "October",
-  11: "November",
-  12: "December"
-}
+const displayMonth = [];
 const chart = document.getElementById('seasonality');
 chart.config = {
   options: {
     timeScale: {
-      tickMarkFormatter: (time, tickMarkType, locale) => {
-        if(!monthCount[time.month]) {
-          monthCount[time.month] = 1;
-          return months[time.month];
+      tickMarkFormatter: (date) => {
```

Comment:
Let's change params to https://tradingview.github.io/lightweight-charts/docs/api#tickmarkformatter

Reply:
Done!